apps/mn-original-populasjon/src/main/java/no/nav/registre/testnorge/originalpopulasjon/consumer/command/GetStatistikkCommand.java

```diff
@@ -0,0 +1,30 @@
+package no.nav.registre.testnorge.originalpopulasjon.consumer.command;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpHeaders;
+import org.springframework.web.reactive.function.client.WebClient;
+
+import java.util.concurrent.Callable;
+
+import no.nav.registre.testnorge.libs.dto.statistikkservice.v1.StatistikkDTO;
+import no.nav.registre.testnorge.libs.dto.statistikkservice.v1.StatistikkType;
+
+@RequiredArgsConstructor
+public class GetStatistikkCommand implements Callable<StatistikkDTO> {
+    private final WebClient webClient;
+    private final StatistikkType type;
+    private final String token;
+
+    @Override
+    public StatistikkDTO call() {
+        return webClient.get()
```

Comment:
Kanskje legge try/catch rundt kallet for feilhåndtering og logging?

Reply:
Ikke med mindre jeg kan håntere feilen. Bedre å bare la systemet plukket opp feilen slik at de ikke blir doublet opp med feil.